spec/takeaway_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) {described_class.new}
+  
+  it 'should return a list of dishes and their respective prices' do
+    expect(takeaway.menu).to eq({:dish1=>1,:dish2=>2,:dish3=>3,:dish4=>4})
```

Comment:
Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)